docs/learn/learn-comparison-ethereum-2.md

```diff
@@ -124,6 +124,31 @@ as long as the logic compiles to Wasm and the shard provides an "execute block"
 validators. Polkadot has the Substrate development framework that allows full spectrum composability
 with a suite of modules that can be configured, composed, and extended to develop a chain's STF.
 
+## Rollups
+
+Rollups, a layer 2 scaling solution, has been talked about for some time, and their potential to scale Ethereum. A layer 2 network of nodes would be responsible for "rolling up" transactions together by batching them before publishing them to the main Ethereum chain. A rolled-up transaction could include 10,000 transactions. Rollups increase TPS and can reduce transaction fees significantly. When it comes to implementing it, there are a few different ways to approach the problem.
+
+### Optimistic Rollups
+
+Using an "innocent until proven guilty" approach, optimistic rollups will accept transaction commitments to the network while validating for errors and security issues. If invalid, validators can submit fraudulent commitment proof (fraud-proofs). These proofs will be used to reject fraudulent transactions. This way, transaction results along with proof can be stored on the main network, but transaction data can be stored elsewhere. Optimistic rollups can improve transaction scalability 10-100x, and write transactions to Ethereum as [`calldata`](https://ethereum.stackexchange.com/questions/52989/what-is-calldata), reducing gas fees significantly.
```

Comment:
'If invalid' sentence grammar is off.

'10-100x' needs a source.

Reply:
Source is ETH docs https://ethereum.org/en/developers/docs/scaling/optimistic-rollups/ I've added it to the resource section